Resources/Prototypes/Catalog/uplink_catalog.yml

```diff
@@ -311,7 +311,7 @@
   description: uplink-exploding-syndicate-bomb-desc
   productEntity: SyndicateBomb
   cost:
-    Telecrystal: 11
+    Telecrystal: 14 # DV - was 11, Syndicate bomb was way too powerful for the 11 TC cost
```

Comment:
not really sure what this does. the Syndie Bomb is already 11 TC which locks you out of any meaningful Syndie loadouts already, this just further locks you out of any meaningful loadouts but doesn't address the issue that this is `too strong`